src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap

```diff
@@ -273,91 +273,83 @@ exports[`RefinementList rendering with facets from search 1`] = `
           autoCapitalize="off"
           autoComplete="off"
           autoCorrect="off"
+          autoFocus={false}
           className="ais-SearchBox-input"
           disabled={false}
-          maxLength="512"
+          maxLength={512}
           onChange={[Function]}
           placeholder="Search"
-          spellCheck="false"
+          spellCheck={false}
           type="search"
+          value=""
         />
         <button
           className="ais-SearchBox-submit"
+          dangerouslySetInnerHTML={
+            Object {
+              "__html": Object {
```

Comment:
this surprises me 🤔 It probably shouldn't be an object here

Reply:
I return the object from the template in the test: https://github.com/algolia/instantsearch.js/blob/f2f903b51e876d5393331896e6de0d9974ea1a4c/src/components/RefinementList/__tests__/RefinementList-test.js#L263-L268

Reply:
would make more sense to return something that would be as real life? ie. a string and not the whole object?

Reply:
You're right. I'll remove them.